Sleep Tracker/SleepTracker/Helpers/RoundedCorner .swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+/// A path consisting of a radius and angles that can be displayed in custom views.
```

Comment:
Maybe smth like this?
```suggestion
/// A shape with custom-rounded corners.
```

Reply:
fixed